src/Init/Renderer/Network.js

```diff
@@ -0,0 +1,9 @@
+import { ipcRenderer } from 'electron'
+
+import * as Network from '../../Actions/Network'
```

Comment:
Generally it's best to avoid `import * as ...` or `import Network` type imports in favor of `import { setInterface } from ...` because the latter allows for much smaller builds due to tree shaking.